HW02/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_02_t_01.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+lines = []
+File.open('log.txt') do |review_file|
+  lines = review_file.readlines
+end
```

Comment:
You can use one line syntax 
```suggestion
lines = File.readlines('log.txt')
```